source/documentation/deploying_apps/index.erb

```diff
@@ -159,6 +159,14 @@ You must create a [network policy](https://cli.cloudfoundry.org/en-US/cf/add-net
 cf add-network-policy PUBLIC_APPNAME --destination-app PRIVATE_APPNAME --protocol tcp --port 8080
 ```
 
+#### Recreate the network policy (optional)
+
+If you are using [version 2.0 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v2) and a [blue-green deployment tool](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) to reduce downtime, you must recreate the network policy every time you push your apps.
+
+This is because every time you push your apps, the blue-green deployment creates a new version of the public app, so the network policy will no longer work.
+
+[Version 3.0 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v3) deploys your app with zero downtime. You do not need to recreate the network policy every time you push your apps.
```

Comment:
I wonder if this section would be better to either not mention the reasons for blue-green deployment tools, or explictly call them out.

> If you are using [version 2.0 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v2) and a [blue-green deployment tool](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html), you must recreate the network policy every time you push your apps.
>
> This is because every time you push your apps with the V2 API, the blue-green deployment creates a new version of the public app, so the network policy will no longer work.
>
> [Version 3.0 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v3) deploys your app with zero downtime. You do not need to recreate the network policy every time you push your apps.

:-/